docs/Appendix-J.rst

```diff
@@ -1,102 +1,147 @@
+.. _running_paie:
 
-Appendix - J Transferring Deployement Container to New Host
-===========================================================
-**Stil in Development**
+Appendix - J Running the PowerAI Enterprise Software Install Module
+===================================================================
 
-TODO: general description
+Overview
+--------
+The POWER-Up software can run on OpenPOWER or x_86 architecture.
 
-Save Container Files
---------------------
+The PowerAI Enterprise Software Install Module provides for rapid installation of the PowerAI Enterprise software to a cluster of Power8 or Power9 servers.
```

Comment:
I'm not sure how official we need to be here, but I believe "POWER8" and "POWER9" should be written in all caps.

Reply:
Yep